build.gradle

```diff
@@ -156,7 +156,7 @@ subprojects {
   }
 
   dependencies {
-    testCompile 'junit:junit:4.11', 'org.hamcrest:hamcrest-library:1.3', 'org.mockito:mockito-core:1.9.5'
+    testCompile 'junit:junit:4.11', 'org.hamcrest:hamcrest-library:1.3', 'org.mockito:mockito-core:1.9.5', 'org.slf4j:slf4j-simple:1.7.7'
```

Comment:
Is adding the dependency to `testRuntime` an issue with IDEs? If not, that's the proper scope to use.